data/ffg-xws.json

```diff
@@ -398,5 +398,82 @@
     "394": "protonbombs",
     "395": "proximitymines",
     "396": "seismiccharges"
+  },
+  "factions": { "1": "rebelalliance", "2": "galacticempire", "3": "scumandvillainy" },
+  "ships": {
+    "1": "modifiedyt1300lightfreighter",
+    "3": "starviperclassattackplatform",
+    "4": "scurrgh6bomber",
+    "5": "yt2400lightfreighter",
+    "6": "auzituckgunship",
+    "7": "kihraxzfighter",
+    "8": "sheathipedeclassshuttle",
+    "9": "quadrijettransferspacetug",
+    "10": "firesprayclasspatrolcraft",
+    "11": "tielnfighter",
+    "12": "btla4ywing",
+    "13": "tieadvancedx1",
+    "14": "alphaclassstarwing",
+    "15": "ut60duwing",
+    "16": "tieskstriker",
+    "17": "asf01bwing",
+    "18": "tieddefender",
+    "19": "tiesabomber",
+    "20": "tiecapunisher",
+    "21": "aggressorassaultfighter",
+    "22": "g1astarfighter",
+    "23": "vcx100lightfreighter",
+    "24": "yv666lightfreighter",
+    "25": "tieadvancedv1",
+    "26": "lambdaclasst4ashuttle",
+    "27": "tiephphantom",
+    "28": "vt49decimator",
+    "29": "tieagaggressor",
+    "30": "btls8kwing",
+    "31": "arc170starfighter",
+    "32": "attackshuttle",
+    "33": "t65xwing",
+    "34": "hwk290lightfreighter",
+    "35": "rz1awing",
+    "36": "fangfighter",
+    "38": "z95af4headhunter",
+    "39": "m12lkimogilafighter",
+    "40": "ewing",
+    "41": "tieininterceptor",
+    "42": "lancerclasspursuitcraft",
+    "43": "tiereaper",
+    "44": "m3ainterceptor",
+    "45": "jumpmaster5000",
+    "47": "customizedyt1300lightfreighter",
+    "48": "escapecraft"
+  },
+  "actions": {
+    "1": "boost",
+    "2": "focus",
+    "3": "evade",
+    "4": "lock",
+    "5": "barrelroll",
+    "6": "reinforce",
+    "7": "cloak",
+    "8": "coordinate",
+    "9": "calculate",
+    "10": "jam",
+    "12": "reload",
+    "13": "slam",
+    "14": "rotatearc"
+  },
+  "stats": {
+    "1": "agility",
+    "2": "hull",
+    "3": "shields",
+    "4": "force",
+    "7": "charge",
+    "8": "dualturretarc",
+    "9": "fullfrontarc",
+    "10": "frontarc",
+    "11": "frontbullseyearc",
+    "12": "mobilearc",
```

Comment:
Interesting that FFG still calls it `mobile arc`. We call it `Single Turret Arc` in xwing-data, which I believe is what they call it in the rules reference.

Reply:
Yep, I just checked the rules reference and will update these arcs names

Reply:
And it appears that I had changed the source after running the script initially. But that has been updated now.